### ISSUE TITLE
Login show/hide password example - icons missing aria-hidden

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#3876]
The `fa-eye` and `fa-eye-slash` icons on the login examples page are missing `aria-hidden="true"`

https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/Login/examples/Login.md
